ee/modules/610-service-with-healthchecks/crds/doc-ru-servicewithhealthchecks.yaml

```
@@ -0,0 +1,414 @@
+spec:
+  versions:
+    - name: v1alpha1
+      schema:
+        openAPIV3Schema:
+          description: Модуль ServiceWithHealthchecks
+          properties:
+            spec:
+              description: ServiceWithHealthchecks — схема для API `servicewithhealthchecks`
+              properties:
+                healthcheck:
+                  properties:
+                    initialDelaySeconds:
+                      description: |-
+                        Время в секундах после страта пода, по истечении которого запускаются пробы. По умолчанию 10.
+                    periodSeconds:
+                      description: |-
+                        Период в секундах для повторных запусков проб. По умолчанию 10.
+                    probes:
+                      items:
+                        properties:
+                          failureThreshold:
+                            description: |-
+                              Минимальное количество последовательных неудач, после которых проба будет считаться неудачной.
+                              
+                              По умолчанию равно 3. Минимальное значение - 1.
+                          timeoutSeconds:
+                            description: Число секунд, по истечении которых проверка завершится по таймауту. По умолчанию — 1 секунда. Минимальное значение — 1.
+                          successThreshold:
+                            description: |-
+                              Минимальное количество последовательных успешных проверок, после которых проверка считается успешной после предыдущих неудач. По умолчанию — 1. Должно быть 1 для проверки живучести (liveness) и запуска (startup). Минимальное значение — 1.
+                          mode:
+                            description: |-
+                              Режим для проверки. Принимает одно из значений: HTTP, TCP, PostgreSQL.
```

```suggestion
                              Режим для проверки. Принимает одно из значений: HTTP, TCP, PostgreSQL.
```
Don't need the "Принимает одно из значений" if there is enum in OpenAPI.